frontend/model/state.js

```diff
@@ -347,33 +347,31 @@ const getters = {
     return monthstamp => groupIncomeDistribution({ state, getters, monthstamp, adjusted: true })
   },
   ourPayments (state, getters) {
+    // Payments relative to the current month only
     const currency = currencies[getters.groupSettings.mincomeCurrency]
     const ourUsername = getters.ourUsername
     const cMonthstamp = currentMonthstamp()
     const pDate = dateFromMonthstamp(cMonthstamp)
     const dueIn = lastDayOfMonth(pDate)
     const monthlyPayments = getters.groupMonthlyPayments
     const allPayments = getters.currentGroupState.payments
+    const thisMonthPayments = monthlyPayments[cMonthstamp]
+    const paymentsFrom = thisMonthPayments && thisMonthPayments.paymentsFrom
```

Comment:
This would be the perfect spot to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) to get rid of the `thisMonthPayments` variable, except that it requires [a slightly newer Rollup version](https://github.com/rollup/rollup/issues/3469).

Reply:
A few months ago I tried to setup optional chaining in this project but couldn't make it work. Some error related with rollup and grunt (as you described). If you find a way to do it, please @ me in the MR so that I can see how it was done!

Reply:
Ah good to know, thanks 👍